_install-and-configure/install-opensearch/index.md

```diff
@@ -102,9 +102,9 @@ Do not declare the same JVM options in multiple locations because it can result
 
 ## Important system properties
 
-OpenSearch has a number of system properties that you can specify in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS` using `-D` command line argument notation, for example:
-
-- `opensearch.xcontent.string.length.max=<value>`
-
-By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.
+OpenSearch has a number of system properties, listed in the following table, that you can specify in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS` using `-D` command line argument notation.
 
+Property | Description
+:---------- | :-------- 
+`opensearch.xcontent.string.length.max=<value>`  |	By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.  | 
```

Comment:
Is this contradictory?

"OpenSearch does not impose any limits on the maximum length of the JSON string fields."

and

"(the maximum is 2,147,483,647)"

Or are these statements referring to two different maximums?

Reply:
It's referring to the reasonable limit max